app/react/utils/specs/api.spec.js

```diff
@@ -14,6 +14,23 @@ import loadingBar from 'app/App/LoadingProgressBar';
 import * as notifyActions from 'app/Notifications/actions/notificationsActions';
 
 describe('api', () => {
+  const validationErrorResponse = {
+    status: 422,
+    body: {
+      error: 'validation failed',
+      validations: [
+        {
+          dataPath: ".metadata['prop1']",
+          message: 'should be string',
+        },
+        {
+          dataPath: ".metadata['prop2']",
+          message: 'is too longer',
```

Comment:
Please fix this English error: "is too long".  "Longer" is a comparison operator.